src/views/postpage.tsx

```diff
@@ -366,7 +366,7 @@ const PostWithCommentTree: Component = () => {
             >
                 <ProfileZone userInfo={threadInfo()!.tryGetStatus()!.account} />
             </Show>
-            <div class="grow flex flex-col justify-start">
+            <div class="flex-grow max-w-4xl flex flex-col justify-start">
```

Comment:
It'd be nice to make this a component that could be reused in multiple places, but that can come later.